lib/apple.js

```diff
@@ -11,12 +11,53 @@ var errorMap = {
 	21008: 'This receipt is a production receipt, but it was sent to the sandbox service for verification.'
 };
 
+var config = null;
 var sandboxHost = 'sandbox.itunes.apple.com';
 var liveHost = 'buy.itunes.apple.com';
 var path = '/verifyReceipt';
 
+function isValidConfigKey(key) {
+    return key.match(/^apple/);
+}
+
+module.exports.readConfig = function (configIn) {
+	if (!configIn) {
+		// no apple iap or password not required
+		return;
+	}
+    config = {};
+    var config_value_set = false;
+    Object.keys(configIn).forEach(function (key) {
+        if (isValidConfigKey(key)) {
+            config[key] = configIn[key];
+            config_value_set = true;
+        }
+    });
+
+    if (!config_value_set) {
+        config = null;
+        return;
+    }
+};
+
+module.exports.setup = function (cb) {
+	if (!config || !config.applePassword) {
+
+		if (process.env.APPLE_IAP_PASSWORD) {
+            config = config || {};
```

Comment:
Great. Thanks.
Yeah. I usually use tabs in my editor.

Thanks for the PR by the way :-)